assets/js/load_more.js

```diff
@@ -0,0 +1,32 @@
+jQuery(document).ready(function () {
+  initLoadMore();
+})
+
+function initLoadMore() {
+  jQuery('.jtf-load-more').on('click', function (e) {
+    e.preventDefault();
+    var $loadmore = jQuery(this),
+      link = $loadmore.attr('href'),
+      selector = $loadmore.attr('data-selector'),
+      container = $loadmore.attr('data-container') || selector;
+    jQuery.post(
+      link,
+      {
+        'jtf_selector': selector,
+        'jtf_load_more': true
+      },
+      function (data) {
+        if (!jQuery.trim(data)) {
+          $loadmore.hide();
+        }
+        jQuery(container).append(data);
+        var currentpage = link.match(/page\/([0-9]+)/)[1];
+        currentpage = ++currentpage;
```

Comment:
weird code... you don't need to equal to something when use ++.

and you can use it directly in the line below